packages/extension-driver-bq/src/lib/bqDataSource.ts

```diff
@@ -42,16 +60,63 @@ export class BQDataSource extends DataSource<any, BQOptions> {
     }
   }
 
+  public override async export({
+    sql: statement,
+    profileName,
+    directory,
+  }: ExportOptions): Promise<void> {
+    // Use "EXPORT DATA" statement to store query result in the GCS bucket {specified in profile.cache.bucketName} temperately, and will remove the stored data after download.
```

Comment:
misspelling `temperately` -> `temporarily`

Reply:
fix in [052be](https://github.com/Canner/vulcan-sql/pull/166/commits/052bea7465fd0c1803e82324c29e6dd3f45bb461)

Reply:
Rewinded [052be](https://github.com/Canner/vulcan-sql/pull/166/commits/052bea7465fd0c1803e82324c29e6dd3f45bb461) to the previous one